### PR TITLE
Wallet update

### PR DESCRIPTION
**Pull request by laboon**
* Update warning about new encryption
* Remove Enzyme wallet
* Clarify that polkadot support can't help with non-Parity wallets
* Add Crypto.com

### CODE REVIEW

**Comment by laboon** [comment]
@SuragSheth please review and add anything necessary

File: docs/build-wallets.md [hunk 1]
```diff
@@ -39,12 +37,12 @@ application.
...
 | [MetaMask](https://metamask.io/index.html)                         | Building          | MetaMask     | IOS, Android, and browser extension    |
 | [Speckle](https://github.com/GetSpeckle/speckle-browser-extension) | Building          | Speckle      | Browser extension                      |
-| [Enzyme](https://getenzyme.dev/)                                   | Building          | BlockXLabs   | Browser extension                      |
 | [Ledger App](https://zondax.ch/kusama.html#overview)               | Building          | Zondax       | Hardware                               |
 | [KodaDot](https://kodadot.netlify.app/#/accounts)                  | Building          | Geefu        | Browser                                |
 | [Subwallet](https://github.com/yxf/subwallet)                      | Building          | yxf          | CLI Wallet                             |
 | [Fearless](https://soramitsu.co.jp/fearless)                       | Building          | Soramitsu    | IOS and Android                        |
+| [Crypto.com App](https://crypto.com/en/index.html)                 | Live              | Crypto.com   | IOS and Android                        |
```
Comment (krichard410) [inline docs/build-wallets.md]: Just a nitpick, but could the all the Live wallets be clustered together and the building ones clustered? I think it makes the table more organized.

Comment (laboon) [reply docs/build-wallets.md]: Good idea!

File: docs/build-wallets.md [hunk 2]
```diff
@@ -8,21 +8,19 @@ sidebar_label: Polkadot Wallets
...
-wallets, including the polkadot.js extension. This is due to an enhanced encryption method, which is
-also noticeable in a slight delay when encrypting/decrypting your wallet. The next version of the
-polkadot.js extension, which will be released soon, will support this new encryption. Other wallet
-software will also need to update to use the latest version of the Polkadot JS API before they will
-be able to use these wallets.
+## Encryption Enhancement
 
-If you are unable to load a JSON file in a wallet, try loading it in the Polkadot-JS web
-application.
+Some recently generated wallets (JSON files) cannot be imported (restored) into older wallet
```
Comment (krichard410) [inline docs/build-wallets.md]:
```suggestion
Some recently generated wallets (`JSON` files) cannot be imported (restored) into older wallet
```
here is JSON supposed to be `json` or `JSON`?

File: docs/build-wallets.md [hunk 3]
```diff
@@ -8,21 +8,19 @@ sidebar_label: Polkadot Wallets
 
 The integration of a [wallet](https://wiki.polkadot.network/docs/en/glossary#wallet) with Polkadot
 allows for simple and easy access to private keys and signing transactions. Below are some wallets
-that support Polkadot along with their development statuses. Note that inclusion does not
-necessarily imply endorsement of that wallet.
+that support Polkadot along with their development statuses.
 
-## Encryption Enhancement
+Note that inclusion does not necessarily imply endorsement of that wallet. Polkadot support can
+provide assistance with issues related to Polkadot-JS, the Polkadot.js plugin, or Parity Signer. For
```
Comment (krichard410) [inline docs/build-wallets.md]:
```suggestion
provide assistance with issues related to Polkadot-JS, the Polkadot{.js} plugin, or Parity Signer. For
```
I believe that's the formal name according to the extension on the chrome shop

File: docs/build-wallets.md [hunk 4]
```diff
@@ -8,21 +8,19 @@ sidebar_label: Polkadot Wallets
...
-polkadot.js extension, which will be released soon, will support this new encryption. Other wallet
-software will also need to update to use the latest version of the Polkadot JS API before they will
-be able to use these wallets.
+## Encryption Enhancement
 
-If you are unable to load a JSON file in a wallet, try loading it in the Polkadot-JS web
-application.
+Some recently generated wallets (JSON files) cannot be imported (restored) into older wallet
+software. This is due to an enhanced encryption method, which is also noticeable in a slight delay
+when encrypting/decrypting your wallet. If you are unable to load a JSON file, please ensure that
```
Comment (krichard410) [inline docs/build-wallets.md]:
```suggestion
when encrypting/decrypting your wallet. If you are unable to load a `JSON` file, please ensure that
```

File: docs/build-wallets.md [hunk 5]
```diff
@@ -8,21 +8,19 @@ sidebar_label: Polkadot Wallets
...
+you are using the latest version of the wallet software. If you are still unable to load it, ensure
+that the wallet software is using the latest version of the Polkadot API.
```
Comment (krichard410) [inline docs/build-wallets.md]: are there instructions somewhere for updating this?

**Review by krichard410 — APPROVED** [review]
This looks great! I left a few comments

File: docs/build-wallets.md (hunk 2 above)
Comment (laboon) [reply docs/build-wallets.md]: JSON is correct, it is an acryonym (JavaScript Object Notation) - see https://en.wikipedia.org/wiki/JSON

File: docs/build-wallets.md (hunk 5 above)
Comment (laboon) [reply docs/build-wallets.md]: not really instructions _per se_, but I included a link to the Polkadot API page

File: docs/build-wallets.md (hunk 1 above)
Comment (laboon) [reply docs/build-wallets.md]: done

File: docs/build-wallets.md (hunk 2 above)
Comment (krichard410) [inline docs/build-wallets.md]:
```suggestion
Some recently generated wallets (`JSON` files) cannot be imported (restored) into older wallet
```

Comment (laboon) [reply docs/build-wallets.md]: fixed

**Pull request merged**